components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -125,6 +139,10 @@ define([
             $super();
 
             this.generateMenuBtn();
+            if (this.getNode().getLifeStatus() == "alive" || this.getNode().getLifeStatus() == "deceased") {
+                this.generateAliveWell();
+            }
+            this.getBoxOnHover().attr({'height': PedigreeEditorParameters.attributes.personHoverBoxRadius * 2 + this.getLabelsOffset()});
```

Comment:
Made to happen always by intension. By default we generate extended hover box because the default live status is 'alive'. When live status changes to anything that's is not alive or deceased we need this line to shrink hover box. If life status changes again to alive we need this line to extend hover box.